README.md

```diff
@@ -72,104 +83,111 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
+- Set default shortcut to Ctrl+Win+Alt+V. (This was a hotfix for 0.67)
+- Update icons. Thanks [@niels9001](https://github.com/niels9001)!
 
-### PowerToys Run
+### PowerRename
 
-- Show Steam (steam://open/) shortcuts in the Program plugin. 
-- Localize paths of Program plugin results. Thanks [@htcfreek](https://github.com/htcfreek)!
-- Improved stability of the code used to get the localized names and paths. Thanks [@htcfreek](https://github.com/htcfreek)!
-- Support MSC and CPL files in "Run command" results of Program plugin. Thanks [@htcfreek](https://github.com/htcfreek)!
-- Added missing MSC and CPL settings to the results of Windows Settings plugin. Thanks [@htcfreek](https://github.com/htcfreek)!
-- System plugin: Setting for separate "Open/Empty Recycle bin" results or single result with context menu. (This was implemented based on user feedback for a change in the last build.) Thanks [@htcfreek](https://github.com/htcfreek)!
-- Add support for log2 and log10 in Calculator plugin. Thanks [@RickLuiken](https://github.com/RickLuiken)!
-- Removed the TimeZone plugin.
-- Fix the crash when loading thumbnail for PDF files. (This was a hotfix for 0.67)
+- Show PowerRename in directory background context menu.
+- Fix the crash on clicking Select/UnselectAll checkbox while showing only files to be renamed.
+- Improve performance on populating Renamed items when many items are being renamed.
 
-### Shortcut Guide
+### PowerToys Run
 
-- Added: Dismiss Shortcut Guide with mouse click. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+- Add setting to disable thumbnails generation for files. (This was a hotfix for 0.67)
+- Calculator plugin - handle implied multiplication expressions. Thanks [@jjavierdguezas](https://github.com/jjavierdguezas)!
+- Fix Calculator plugin unit tests to respect decimal separator locale. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+- Fix crashing issue caused by thumbnail image loading.
+- Date & Time plugin - Add filename-compatible date & time format. Thanks [@Picazsoo](https://github.com/Picazsoo)!
+- Improve plugin loading error. Thanks [@htcfreek](https://github.com/htcfreek)!
 
 ### Quick Accent
 
-- Added Lithuanian characters. Thanks [@saulens22](https://github.com/saulens22)!
-- Added additional (Chinese) characters. Thanks [@ChaseKnowlden](https://github.com/ChaseKnowlden)!
+- Fix existing and add missing Hebrew and Pinyin characters. Thanks [@stevenlele](https://github.com/stevenlele)!
+
+### Registry Preview
+
+- New utility: Registry Preview is a utility to visualize and edit Windows Registry files. Thanks [@randyrants](https://github.com/randyrants)!
+- Fix resizing and moving logic. Thanks [@randyrants](https://github.com/randyrants)!
+- UI improvements. Thanks [@niels9001](https://github.com/niels9001)!
+- Update icon. Thanks [@niels9001](https://github.com/niels9001)!
+- Remove read-only checkbox. Thanks [@randyrants](https://github.com/randyrants)!
```

Comment:
These are not fixes if this is the first release
```suggestion

```

Reply:
Then it's a joint achievement indeed.

Reply:
My intention here was what @htcfreek said. To point out the help with shipping the utility.

Reply:
Nevertheless, release notes are to show changes that a user could notice. These are not.

Reply:
I just went through some older release notes and update this PR according to how it was being done before.